app/web_framework/cluster_master_application.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, cluster_master):
         api_v2 = [
             RouteNode(r'metrics', _MetricsHandler),
             RouteNode(r'version', _VersionHandler),
-            RouteNode(r'builds', _BuildsHandler).add_children([
+            RouteNode(r'builds', _BuildsPaginatedHandler).add_children([
```

Comment:
I think that's fine. It would help make it clear just from looking at the routenodes which endpoints had changed for v2 and which were still falling back to old logic.